.github/ISSUE_TEMPLATE/RFC_report.md

```diff
@@ -0,0 +1,87 @@
+---
+name: RFC template
+about: request for comments
+title: "[RFC]"
+labels: 'RFC'
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the RFC in the Title above -->
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
```

Comment:
I am really not sure what is the use of Start Date. Should there be an _End Date_?

I think maybe this info is not suitable to our needs. If we are done with discussing a RFC (whether giving it a PASS or REJECT), we can just close the issue and move on implementing and submitting PRs.

Reply:
My understanding is, when we want to collect those RFC to be a document, the reader is able to know the history.

Reply:
Ok, make sense.